modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/ImagePicker/ImagePicker.es.js

```diff
@@ -283,6 +282,8 @@ const Main = ({
 	const prevEditingLanguageId = usePrevious(editingLanguageId);
 
 	if (prevEditingLanguageId !== editingLanguageId && localizable) {
+		const localizedValue = otherProps.localizedValue;
+
```

Comment:
Hello, @MirnaGama.

You could only add localizedValue on:

`return (
		<FieldBase
			{...otherProps}
			localizedValue={localizedValue}
`
instead of creating this  const: 
`const localizedValue = otherProps.localizedValue; `.

But as this PR is high priority, feel free to do it in another time.

Reply:
@alinedoleron done ! 😃